addons/frag/XEH_PREP.hpp

```diff
@@ -1,23 +1,35 @@
+// dev
+#ifdef DEBUG_MODE_DRAW
+PREP(dev_fired);
+PREP(dev_trackHitBox);
+PREP(dev_sphereDraw);
+PREP(dev_addRound);
+PREP(dev_trackObj);
+PREP(dev_drawTrace);
+PREP(dev_clearTraces);
+PREP(dev_switchUnitHandle);
+#endif
+PREP(dev_fragCalcDump);
 PREP(dev_debugAmmo);
 
-PREP(doSpall);
-PREP(fired);
-PREP(frago);
-PREP(spallTrack);
+// Base
+PREP(initRound);
```

Comment:
Make a single block of PREP macros, ordered alphabetically.
Might as well bring this up to date with other components.

Reply:
Should I drop the includes as well (to be inline) or are they alright to skip when not dev compiled?

Reply:
As you've changed is what I had in mind.